README.md

```diff
@@ -141,6 +141,57 @@ If you need to change the Judoscale integration configuration, you can pass a di
 judoscale_celery(broker, extra_config={"LOG_LEVEL": "DEBUG"})
 ```
 
+# Using Judoscale with RQ
+
+Install Judoscale for Celery with:
+
+```sh
+$ pip install 'judoscale[rq]'
+```
+
+Judoscale can automatically scale the number of RQ workers based on the queue latency (the age of the oldest pending task in the queue).
+
+## Configuring the application
```

Comment:
I think they're both required, yes. RQ does not come with a canonical "main" class that coordinates the details about queues/workers/configuration/etc, like Celery.

Instead, Queues, Workers, etc are all separate classes. `judoscale_rq()` is called in the app factory (`create_app()`), which would start the reporter for the worker if the worker created and imported the app, but the worker does not _need_ the `app`, strictly speaking.

You could imagine a scenario with an app that enqueues tasks to ping some API endpoint. All the data needed by the task is passed in as arguments. That task does not need to depend on the app context and the author of such an app might choose to not create/import the app at all. This would mean the reporter does not get started for the worker(s).

On the other hand, I'm not sure how common the above scenario is in practise. Maybe it's such a low percentage share of the apps that don't import the app in the worker that we could impose a requirement to always import the app. I'm happy to revisit the current approach.

Reply:
@adamlogic I realised that we can actually do away with the separate configuration of the app and workers — it's not needed after all. Thanks for challenging this!